dojo/api_v2/views.py

```diff
@@ -557,7 +559,20 @@ class DojoMetaViewSet(mixins.ListModelMixin,
     filter_backends = (DjangoFilterBackend,)
     filter_fields = ('id', 'product', 'endpoint', 'name', 'finding')
 
-
+def _product_expand_decorator():
+    return swagger_auto_schema(
+    responses={status.HTTP_200_OK: serializers.ProductSerializer},
+    manual_parameters=[
+        openapi.Parameter(
+            name="expand", 
+            in_=openapi.IN_QUERY, 
+            description="Fields to expand based on their relation", 
```

Comment:
Really cool to be able to document it this way, nice finding ! We could give more info on the `description` as it might be hard to understand without context but I'm unable to find something better ^^.